tests/unit/components/error-display-test.js

```diff
@@ -0,0 +1,100 @@
+import { moduleForComponent, test } from 'ember-qunit';
+
+moduleForComponent('error-display', 'Unit | Component | error display', {
+  unit: true
+});
+
+test('properties have default values', function(assert) {
+  assert.expect(1);
+
+  const expected = {
+    content:     null,
+    showDetails: false
+  };
+
+  const component = this.subject();
+
+  const actual = {
+    content:     component.get('content'),
+    showDetails: component.get('showDetails'),
+  };
+
+  assert.deepEqual(actual, expected, 'default values are correct');
+});
+
+// test('`toggleDetails` gets triggered on click', function(assert) {
```

Comment:
It used to be a passing test until I had to deal with translation again. I can remove it.